core/federated/federate.c

```diff
@@ -1966,23 +1994,25 @@ void lf_connect_to_rti(const char* hostname, int port) {
 #endif
 
         // Send the message type first.
-        unsigned char buffer[4];
+        unsigned char buffer[5];
         buffer[0] = MSG_TYPE_FED_IDS;
         // Next send the federate ID.
         if (_lf_my_fed_id > UINT16_MAX) {
             lf_print_error_and_exit("Too many federates! More than %d.", UINT16_MAX);
         }
         encode_uint16((uint16_t)_lf_my_fed_id, &buffer[1]);
+        // Next send the federate type (persistent or transient)
+        buffer[1 + sizeof(uint16_t)] = _fed.is_transient? 1 : 0;
```

Comment:
This change to the `MSG_TYPE_FED_IDS` needs to be documented in `net_common.h`.

Reply:
Done in [b33b863](https://github.com/lf-lang/reactor-c/pull/358/commits/b33b86304f9b8efc5b165d9b4765667045ed853b).